utils/formatChartDataForNivo.ts

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
I'm not 100% that we should include this or the colors in this package.  I can see the benefit of providing this for our end users, but I'm hesitant to show favoritism to specific chart libraries.  I wonder if we should skip this and the colors, just keep it internal in our app.

Reply:
https://github.com/transform-data/transform-react/pull/4#discussion_r600929693